mkDocs.sh

```diff
@@ -77,3 +99,13 @@ docker run --rm \
 	busybox \
         sh -c 'cp /opt/maven/CHANGELOG.md /opt/maven/target/generated-docs/CHANGELOG_ARLAS-server.md'
 
+echo "=> Check generated documentation"
+if [[ ! -f ${BASEDIR}/target/generated-docs/typescript-doc/classes/_api_.exploreapi.md ]] ; then
+    echo 'File "_api_.exploreapi.md" is not generated, aborting.'
+    exit -1
+fi
+if [[ ! -f ${BASEDIR}/target/generated-docs/reference.md ]] ; then
+    echo 'File "reference.md" is not generated, aborting.'
+    exit -1
+fi
+echo "Ok!"
```

Comment:
Useless `echo` since exit code is traped and call `clean` function iwhich print status code (if `Exit stage = 0`, it's ok ;) )

Reply:
ok